### PR TITLE
Encyclopedia clean up following #9188

### PR DESCRIPTION
**Pull request by Wedge009**
CC: @Gothyoba

### CODE REVIEW

**Comment by Gothyoba** [comment]
Looks good to me, though I’m not the one that needs to review this.

I will still be making a follow-up as a result of lore changes/additions in TDG and the TSG Re-Revision, which will probably fix anything that’s been missed here as well as possibly tweaking some stuff. That should probably wait until at least one of those is merged however.

Thanks!

**Review by knyghtmare — APPROVED** [review]

**Review by Dalas121 — APPROVED** [review]

**Pull request merged**
